.vscode/settings.json

```diff
@@ -2,5 +2,63 @@
      "C_Cpp.clang_format_style": "{ BasedOnStyle: Google, UseTab: Never, IndentWidth: 4, TabWidth: 4, AllowShortIfStatementsOnASingleLine: false, IndentCaseLabels: true, ColumnLimit: 80, AccessModifierOffset: -3, AlignConsecutiveMacros: true }",
      "editor.formatOnSave": true,
      "editor.formatOnType": true,
-     "editor.formatOnPaste": true
+     "editor.formatOnPaste": true,
+     "files.associations": {
+          "cctype": "cpp",
```

Comment:
Hmm, I'm not sure if this file should be modified. @ayaankhan98, @mishraabhinn, thoughts?

Reply:
I don't have an idea about this @Panquesito7

Reply:
Indeed the ".vscode / settings.json" file must not be modified. The CI script returned the file to its original state.

Reply:
@Jxtopher you deleted the file, please revert it back with no changes as it was originally.